v3/src/components/graph/adornments/lsrl/lsrl-adornment-model.ts

```diff
@@ -23,6 +26,33 @@ export const LSRLInstance = types.model("LSRLInstance", {
            isFinite(Number(self.rSquared)) &&
            isFinite(Number(self.slope)) &&
            isFinite(Number(self.sdResiduals))
+  },
+  sumOfSquares(dataConfig: IGraphDataConfigurationModel, layout: IAxisLayout, cellKey: any) {
+    const dataset = dataConfig?.dataset
+    const caseData = dataset?.cases
+    const xAttrID = dataConfig?.attributeID("x") ?? ""
+    const yAttrID = dataConfig?.attributeID("y") ?? ""
+    let sumOfSquares = 0
+    caseData?.forEach((datum: any) => {
+      const fullCaseData = dataConfig?.dataset?.getCase(datum.__id__)
+      if (fullCaseData && dataConfig?.isCaseInSubPlot(cellKey, fullCaseData)) {
+        const x = dataset?.getNumeric(datum.__id__, xAttrID) ?? 0
+        const y = dataset?.getNumeric(datum.__id__, yAttrID) ?? 0
```

Comment:
Defaulting to zero here seems dangerous. It shouldn't ever happen, but if it does, 0 will invalidate the result. I think `NaN` would be preferable.